docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc

```diff
@@ -0,0 +1,477 @@
+    "FlightDelay": false,
+    "DestWeather": "Sunny",
+    "Dest": "Chubu Centrair International Airport",
+    "FlightDelayType": "No Delay",
+    "OriginCountry": "AU",
+    "dayOfWeek": 3,
+    "DistanceKilometers": 7729.461862731618,
+    "timestamp": "2019-10-17T11:12:29",
+    "DestLocation": {
+      "lat": "34.85839844",
+      "lon": "136.8049927"
+    },
+    "DestAirportID": "NGO",
+    "Carrier": "ES-Air",
+    "Cancelled": false,
+    "FlightTimeMin": 454.6742272195069,
+    "Origin": "Adelaide International Airport",
+    "OriginLocation": {
+      "lat": "-34.945",
+      "lon": "138.531006"
+    },
+    "DestRegion": "SE-BD",
+    "OriginAirportID": "ADL",
+    "OriginRegion": "SE-BD",
+    "DestCityName": "Tokoname",
+    "FlightTimeHour": 7.577903786991782,
+    "FlightDelayMin": 0
+  }
+}
+```
+
+{classification-cap} is a supervised machine learning analysis and therefore 
+needs to train on data that contains the ground truth, known as the 
+`dependent_variable`. In this example, the ground truth is available in each 
+document as the actual value of `FlightDelay`.
+
+If your source data consists of some documents that contain a 
+`dependent_variable` and some that do not, the model is trained on the 
+`training_percent` of the documents that contain ground truth. However, 
+predictions are made against all of the data. The current implementation of 
+{classanalysis} supports a single batch analysis for both training and 
+predictions.
+
+
+. Create a {dfanalytics-job}.
++
+--
+Use the {ref}/put-dfanalytics.html[create {dfanalytics-jobs}] API as you can see 
+in the following example:
+
+[source,console]
+--------------------------------------------------
+PUT _ml/data_frame/analytics/model-delayed-flights
+{
+  "source": {
+    "index": [
+      "kibana_sample_data_flights"  <1>
+    ]
+  },
+  "dest": {
+    "index": "df-flight-delayed",  <2>
+    "results_field": "ml" 
+  },
+  "analysis": {
+    "classification": {
+      "dependent_variable": "FlightDelay",  <3>
+      "training_percent": 10  <4>
+    }
+  },
+  "analyzed_fields": {
+    "includes": [],
+    "excludes": [    <5>
+      "Cancelled",
+      "FlightDelayMin",
+      "FlightDelayType"
+    ]
+  },
+  "model_memory_limit": "100mb" <6>
+}
+--------------------------------------------------
+// TEST[skip:setup kibana sample data]
+
+<1> The source index to analyze.
+<2> The index that will contain the results of the analysis; it will consist of 
+a copy of the source index data where each document is annotated with the 
+results.
+<3> Specifies the variable we want to predict with the {classanalysis}.
+<4> Specifies the proportion of data that is used for training the model. In 
+this example we use 10% of the source data. The training data is randomly 
+selected and provide predictions from the remaining 90%. These percentages are 
+approximate. For most of the datasets, even a small proportion of the training 
+data (2-5%) results in good predictions.
```

Comment:
> For most datasets, even a small proportion of the training data...

Can we clarify why you'd consider specifying this?  Is it so that it's faster?  Or is it so that you can use the rest as testing data?

Reply:
Yes, I clarified this. It is important when you need to manually label your dataset with the ground truth.